packages/cli/src/commands/styles/utils/publish.main.ts

```diff
@@ -21,39 +22,71 @@ export const generateToken = (name: string) => {
 export const filterFunctions: NodeFilter[] = [];
 export const designTokenFunctions: DesignTokenFilter[] = [];
 
-const colorDesignTokenizer: DesignTokenFilter = (node: NodeDocument): DesignToken => {
-  colorToHex(node.fills[0].color);
-  const colorToken: DesignToken = {
-    type: 'color' ,
-    name: node.name,
-    token: generateToken(node.name),
-    value: colorToHex(node.fills[0].color)
-  };
-  return colorToken;
-};
 
-const colorFilter: NodeFilter = (data: NodeDocument, designTokenFilterFn: DesignTokenFilter): DesignToken => {
-  if(data.type == 'RECTANGLE') {
-    data.type = 'color';
+export const processStyleNodes = (data: NodeDocument, type: string): DesignToken[] | DesignToken | undefined => {
```

Comment:
can we make `(type: string)` `(type: StyleType)`? That allows us to type check either 'FILL' 'TEXT' 'EFFECT' or 'GRID' and match our case values to `StyleType` types. We wont be able to add any other gibberish for each case :).